### PR TITLE
fix: php 8 deprecation warning from uksort function

### PR DESCRIPTION
**Pull request by UVLabs**
Fixed PHP 8 warning on orbit fox dashboard

Closes https://github.com/Codeinwp/themeisle-companion/issues/735

### CODE REVIEW

**Comment by UVLabs** [comment]
@cristian-ungureanu Do you think we should update the minimum version of PHP required for orbit fox to 7.0 to make use of this operator? Or just implement a different kind of randomiser function. 

I checked the stats and this is what it looks like for PHP 5.6:

image
https://wordpress.org/about/stats/

**Comment by UVLabs** [comment]
> @UVLabs we shouldn't update the PHP version just because of a sorting function. Maybe we can use https://www.php.net/manual/en/function.shuffle.php instead.

@cristian-ungureanu Made the change here

File: core/app/class-orbit-fox-admin.php [hunk 1]
```diff
@@ -454,12 +454,8 @@ public function get_recommended_plugins() {
...
-		uksort(
-			$data,
-			function() {
-				return rand() <=> rand();
-			}
-		);
+
+		shuffle( $data );
 
 		return $data;
```
Comment (cristian-ungureanu) [inline core/app/class-orbit-fox-admin.php:460]: @UVLabs we can return shuffle( $data ) directly.

**Review by cristian-ungureanu — CHANGES_REQUESTED** [review]

File: core/app/class-orbit-fox-admin.php (hunk 1 above)
Comment (UVLabs) [reply core/app/class-orbit-fox-admin.php:460]: @cristian-ungureanu it returns a bool but we need the array

Comment (cristian-ungureanu) [reply core/app/class-orbit-fox-admin.php:460]: I see, thank you for checking

**Review by cristian-ungureanu — APPROVED** [review]

**Comment by irinelenache** [comment]
@UVLabs Tested and the fix works fine, will move it to **Ready to merge** 🚀

**Pull request merged**